src/app/ipv/middleware.ts

```diff
@@ -439,20 +514,33 @@ async function checkFormRadioButtonSelected(req, res, next) {
   } catch (error) {
     return next(error);
   }
-}
+};
 
-async function formHandleUpdateDetailsCheckBox(req, res, next) {
+export const formHandleUpdateDetailsCheckBox: RequestHandler = async (
+  req,
+  res,
+  next,
+): Promise<void> => {
```

Comment:
I think you shouldn't need the return type if you put a type on the `const` itself